src/store/actions.js

```diff
@@ -192,12 +193,15 @@ export default {
 
     commit('setTipping', [contractInstanceV1, contractInstanceV2]);
   },
+
+  /**
+   * @param {{text: String}} options
+   */
   async share(_, options) {
-    await (process.env.IS_CORDOVA
-      ? new Promise((resolve) => window.plugins.socialsharing.shareW3C(
-        options,
-        ({ app }) => app && resolve(),
-      ))
-      : navigator.share(options));
+    await Share.share({
```

Comment:
Why are you always trying to share with the ionic plugin first?

Reply:
You are correct 👀